cpp/src/gandiva/context_helper.cc

```diff
@@ -50,4 +64,16 @@ void gdv_fn_context_set_error_msg(int64_t context_ptr, char const* err_msg) {
       reinterpret_cast<gandiva::ExecutionContext*>(context_ptr);
   (execution_context_ptr)->set_error_msg(err_msg);
 }
+
+uint8_t* gdv_fn_context_arena_malloc(int64_t context_ptr, int32_t size) {
+  gandiva::ExecutionContext* execution_context_ptr =
```

Comment:
By the way, you can write:
```cpp
auto execution_ptr = reinterpret_cast<gandiva::ExecutionContext*>(context_ptr);
```

(a bit less typing involved :-))

Reply:
will fix.